src/util/radixError.ts

```diff
@@ -0,0 +1,12 @@
+type RadixErrorInput = {
+  message: string
+  code?: number
+  details?: {
+    type: string
+  } & any
+  traceId?: string
+}
+
+export const radixError = (error: RadixErrorInput) => error
+
+export type RadixError = ReturnType<typeof radixError>
```

Comment:
this just exports the `RadixErrorInput` type right now - is the idea to build on this later somehow?